### ISSUE TITLE
HTML streaming not working with Remix app

### ISSUE THREAD

**Issue by tom-sherman** [stackblitz/webcontainer-core#966]
### Describe the bug

When I run this locally, page is rendered as HTML with the suspense boundary as "Loading..." then the promise resolves on the server and is streamed to the client, and the "Loading..." changes to "Hello World".

On stackblitz though, no data is sent until the entire response is done (and so you see no "Loading..." state)

### Link to the blitz that caused the error

https://stackblitz.com/edit/remix-starter-updated-t1fani?file=app%2Froutes%2Findex.tsx,app%2Fentry.server.tsx

### Steps to reproduce

1. Open the linked blits
2. Load the index route

### Expected behavior

See a Loading... message, and then this changes to "Data: Hello World"

### Parity with Local

- [X] I have [run the project in my local machine](https://github.com/stackblitz/webcontainer-core/blob/main/repro.md#try-locally) and I could not reproduce the issue.

### Platform

```
Browser name  = Chrome
Full version  = 110.0.0.0
Major version = 110
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 83867546,
  "usedJSHeapSize": 82016534,
  "jsHeapSizeLimit": 4294705152
}
Hash = 927bd077ab89da68c6338c4363cc22e79cccb4ef
```

**Comment by jrvidal** [comment]
:wave: Thanks for the report. You're correct, the preview window does not properly stream the response back. This is something we haven't got around implementing.

We're tracking this internally now (thanks!).

**Comment by tom-sherman** [comment]
@jrvidal I also couldn't get streaming to work when I opened the preview window in a new tab btw

**Comment by jrvidal** [comment]
Yes, that is expected :+1: It's an issue with our networking stack.

**Comment by Nemikolh** [comment]
Hey everyone! We have added initial support for HTTP streaming! :tada:

@tom-sherman, I have tested your example and we can now see `Loading..` before `Data: Hello World!`.

@Rich-Harris, I've tested your PR and it also works!

I'll be on the lookout for performance regressions, so if you notice anything please let me know! :smiley:
